src/core/Group.cpp

```diff
@@ -169,6 +169,22 @@ QString Group::defaultAutoTypeSequence() const
     return m_data.defaultAutoTypeSequence;
 }
 
+QString Group::effectiveAutoTypeSequence() const
+{
+    QString sequence;
+    const Group* grp = this;
+    do {
+        if (grp->autoTypeEnabled() == Group::Disable) {
```

Comment:
//and it segfaults here but not because of that function call, but because of `Group::Disable` (and because `this` is freakin null here!)